backend/src/event/event.service.ts

```diff
@@ -315,7 +334,7 @@ export class EventService {
     const scheduleDTO: ScheduleDTO = {
       startTime: dto.startTime,
       endTime: dto.endTime,
-      availability: AvailabilityEnum.AVAILABLE,
+      availability: AvailabilityEnum.UNAVAILABLE,
```

Comment:
initially, when creating an event, the availability of the schedule should be `UNAVAILABLE`